src/examplePlugins/update_task_template_entities.py

```diff
@@ -81,15 +81,15 @@ def is_valid(sg, logger, args):
     }
 
     # Check our args.
-    for name, checks in args_to_check.iteritems():
+    for name, checks in args_to_check.items():
 
         # Grab the setting's value type.
         value_type = type(args[name])
 
         # We assume unicode and str to be equivalent for these checks because
         # args come back from Django as unicode but are first set by the
         # Registrar as str.
-        if value_type == unicode:
+        if value_type.__name__ == "unicode":
```

Comment:
`instanceof(value, six.text_type)`

is the cleaner way to check for this. When we clean up our code in the long future where Python 2 is not supported anymore finding checks for six.text_type will be super easy.

Reply:
I thought about doing this. There's an inconsistency in the way the argument type validation is done across the different plugins. Most of them use the `type.__name__` like this (look at the values under the `type` key):

https://github.com/shotgunsoftware/shotgunEvents/blob/6d0ec8dd359aaf4131aac161726c822ac776d057/src/examplePlugins/update_timecode_from_frames.py#L62-L66
https://github.com/shotgunsoftware/shotgunEvents/blob/6d0ec8dd359aaf4131aac161726c822ac776d057/src/examplePlugins/update_timecode_from_frames.py#L81-L87

And a couple of them use the type itself:
https://github.com/shotgunsoftware/shotgunEvents/blob/6d0ec8dd359aaf4131aac161726c822ac776d057/src/examplePlugins/update_task_template_entities.py#L67-L81

I decided to do the string compare to keep it consistent with the other ones. I have no problem changing it to the `isinstance` method. But I did want to point this out to see if you think we should change all the other ones to not use the string matching either.

Reply:
Better is the enemy of good. If this is the pattern used in the repo, let's not rock the boat.